### DISCUSSION TITLE
Hide  "Show source" button when printing Mermaid diagram

### DISCUSSION THREAD

**Discussion by winkelement** [outline/outline#4538] [Ideas]
Button "Show source" should not be visible in the print layout for mermaid diagram.
<img width="523" alt="Bildschirmfoto 2022-12-04 um 20 42 06" src="https://user-images.githubusercontent.com/3525191/205511994-3e67ebf0-6c10-425f-bf11-680285107c84.png">

**Comment by tommoor** [comment]
👍 This would also be solve with #4486

**Reply by winkelement** [reply]
That would be an excellent solution as it also prevents the Button from overlapping with the diagram on the page 🎉
